sdk/src/process_instruction.rs

```diff
@@ -309,6 +313,19 @@ pub mod stable_log {
         ic_logger_msg!(logger, "Program log: {}", message);
     }
 
+    /// Log return data as from the program itself.
+    ///
+    /// The general form is:
+    ///
+    /// ```notrust
+    /// "Program return data: <program-generated-in-base64>"
```

Comment:
```suggestion
    /// "Program return data: <program-generated-data-in-base64>"
```

If we end up adding the program id to the return data (cc: https://github.com/solana-labs/solana/pull/19318#discussion_r704627320) then I think it'll be good to include the program id that generated the return data in this log as well

Reply:
Absolutely. Now it included the program_id of the setter.